coderd/notifications/notifications_test.go

```diff
@@ -460,7 +466,8 @@ func TestExpiredLeaseIsRequeued(t *testing.T) {
 
 	// nolint:gocritic // Unit test.
 	ctx := dbauthz.AsSystemRestricted(testutil.Context(t, testutil.WaitSuperLong))
-	_, _, api := coderdtest.NewWithAPI(t, nil)
+	store, _ := dbtestutil.NewDB(t)
+	logger := slogtest.Make(t, nil).Leveled(slog.LevelDebug)
```

Comment:
Do we need to ignore errors on these?

Reply:
I don't believe so.  Does the manager or notifier drop error logs in this test case?

Note that `context.Canceled` is already ignored by default.